src/operator.lisp

```diff
@@ -13,9 +13,9 @@
 @export
 (defparameter *inside-select* nil)
 
-(defmacro define-op ((op-name struct-type &key sql-op-name include-slots) &body body)
+(defmacro define-op ((op-name struct-type &key sql-op-name include-slots (package (find-package :sxql.operator))) &body body)
```

Comment:
It occurs to me that there might not be a use-case for `package` ever being anything other than `:sxql.operator`.  Let me know what you think!